test/canary/scripts/install_controller_helm.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# Deploy ACK Helm Charts
+
+function install_helm_chart() {
+    local service="$1"
+    local oidc_role_arn="$2"
+    local region="$3"
+    local namespace="$4"
+
+    yq w -i helm/values.yaml "serviceAccount.annotations" ""
+    yq w -i helm/values.yaml 'serviceAccount.annotations."eks.amazonaws.com/role-arn"' "$oidc_role_arn"
+    yq w -i helm/values.yaml "aws.region" $region
+
+    kubectl create namespace $namespace
+    kubectl apply -f helm/crds
+    helm install -n $namespace ack-$service-controller --skip-crds helm
```

Comment:
`yq` writes can be replaced with `--set` when doing `helm install`.

e.g.

```bash
helm install -n $namespace ack-$service-controller --skip-crds --set "serviceAccount.annotations"="" --set 'serviceAccount.annotations."eks.amazonaws.com/role-arn"'="$oidc_role_arn" --set "aws.region"="$region" helm
```

Reply:
This is copy pasted as is from the sagemaker canary at the moment. It should be cleaned up and moved to a common repo (test-infra maybe). 
Would it be ok if we did this cleanup post preview and for GA on a larger scale ?